test/test_quantized_conv.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import torch
 import torch.nn.quantized.functional as qF
+from torch.nn.quantized.modules.conv import _conv_output_shape
```

Comment:
is `torch.nn.quantized.modules.conv` considered as public? should we access them through `torch.nn.quantized` namespace?

Reply:
It is public. This is a general PyTorch (and python) structure -- we have `torch.nn.modules.conv` and `torch.quantized.modules.conv` available to public, as well as having shortcuts in the `torch.nn` and `torch.nn.quantized` respectively.